tests/jballerina-unit-test/src/test/resources/test-src/statements/matchstmt/const-pattern.bal

```diff
@@ -405,6 +405,23 @@ function testConstPattern12() {
     assertEquals("OTHER", constPattern12(3));
 }
 
+client class Client {
+    remote function post(string x, string y) returns anydata {
+    }
+}
+
+Client clientEndpoint = new ();
+public function testConstPattern13() returns string {
```

Comment:
Ideally, we should validate something here, right? Shall have `post` return some value, and assert it inside the match, and remove `returns string` here?

Reply:
changed.